Makefile

```diff
@@ -148,25 +144,72 @@ build-test-unit-js:
 		--file ./test/docker-compose.unit-tests-js.yml
 
 .PHONY: build-test-integration
-build-test-integration: build
+build-test-integration:
+	@echo "--- Building integration test images"
 	$(DOCKER_BUILDX_BAKE) \
 		--file ./test/docker-compose.integration-tests.build.yml \
-		python-integration-tests rust-integration-tests
+		python-integration-tests \
+		rust-integration-tests
 
 .PHONY: build-test-e2e
-build-test-e2e: build
-	./pants package ./src/python/e2e-test-runner/e2e_test_runner:pex
+build-test-e2e:
+	@echo "--- Building e2e testing image"
+# Any PEX tagged with `e2e-test-pex` is required for our image. This
+# seems like the most straightforward way of capturing these
+# dependencies at the moment.
+	./pants --tag="e2e-test-pex" package ::
```

Comment:
so today i learned how to comment makefiels correctly without it showing up at runtime 😢